mcenter_cli/parallelm/mcenter_cli/command_mlapp.py

```diff
@@ -0,0 +1,92 @@
+import pprint
+
+from cliff.command import Command
+
+from parallelm.mlapp_directory.mlapp_defs import MLAppVersions
+from parallelm.mlapp_directory.mlapp_defs import MLAppProfileKeywords
+from parallelm.mcenter_cli.upload_mlapp import upload_mlapp
+from parallelm.mcenter_cli.upload_mlapp_v2 import upload_mlapp_v2
+from parallelm.mcenter_cli.delete_mlapp import MLAppDeleteHelper
+from parallelm.mlapp_directory.directory_from_mlapp_builder import DirectoryFromMLAppBuilder
+
+
+class MLAppListCommand(Command):
+    def take_action(self, parsed_args):
+        fmt = "{:<35} {:<35}"
+        s = ""
+        s += fmt.format("Profile Name", "Pattern Name")
+        for profile_info in self.app.mclient.list_ion_profiles():
+            profile_name = profile_info[MLAppProfileKeywords.NAME]
+            pattern_name = profile_info[MLAppProfileKeywords.PATTERN_NAME]
+            if len(s) > 0:
+                s += "\n"
+            s += fmt.format(profile_name, pattern_name)
+        self.app.stdout.write(s)
+
+
+class MLAppInfoCommand(Command):
+    def get_parser(self, prog_name):
+        parser = super(MLAppInfoCommand, self).get_parser(prog_name)
+        parser.add_argument('name', action='store', help='Name of MLApp to query for info')
+        return parser
+
+    def take_action(self, parsed_args):
+        all_profiles = self.app.mclient.list_ion_profiles()
+        mlapp_info = list(filter(lambda x: x["name"] == parsed_args.name, all_profiles))
+        if len(mlapp_info) == 0:
+            raise Exception("Could not find mlapp with name: {}".format(parsed_args.name))
+        if len(mlapp_info) > 1:
+            raise Exception("Found {} mlapps with the same name".format(len(mlapp_info)))
+        mlapp_info = mlapp_info[0]
+        self.app.stdout(pprint.pformat(mlapp_info))
+
+
+class MLAppUploadCommand(Command):
+    def get_parser(self, prog_name):
+        parser = super(MLAppUploadCommand, self).get_parser(prog_name)
+        parser.add_argument('name', action='store', help='Name of MLApp to query for info')
+
+        parser.add_argument('appdir', action='store', help='directory to use for loading the MLApp')
```

Comment:
Please change the first letter in all your help strings to be caps, so as to be consistent across all your command help strings.

Reply:
Done